chamberInfo.py

```diff
@@ -33,3 +33,21 @@
     8:"long",
     9:"long"
     }
+
+chamber_defaults = {
```

Comment:
Let's rename this to a more accurate description.

You're storing parameters for the VFAT preamp, shaper, and comparator, these are applied to each VFAT on a detector.  This is more for the VFAT than the "detector."

Also the defaults are provided at:

https://github.com/cms-gem-daq-project/cmsgemos/blob/generic-amc-release-v3/gempython/tools/vfat_user_functions_uhal.py#L15

Instead let's call this dict **chamber_vfatDACSettings** instead